docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+Configuration and usage
+=======================
+
+Getting around administrator interface
+--------------------------------------
+
+`Edx-enterprise` is integrated with edx-platform Django admin. It can be accessed at
```

Comment:
We also should mention that this admin interface is being considered a provisional tool for administering the enterprise features, and that we intend to build a nicer, full-featured admin portal that runs as an independent application at some point in the future. Thus, we have not spent much time on the UX of these admin tools, nor do we plan to.